pkg/kncloudevents/message_sender_test.go

```diff
@@ -53,10 +53,10 @@ func TestRetryConfigFromDeliverySpec(t *testing.T) {
 			backoffDelay:  "PT2.5S",
 			expectedBackoffDurations: []time.Duration{
 				2500 * time.Millisecond,
-				2500 * time.Millisecond,
-				2500 * time.Millisecond,
-				2500 * time.Millisecond,
-				2500 * time.Millisecond,
+				5000 * time.Millisecond,
```

Comment:
nit: I would have preferred seeing `2 * 2500`, `3 * 2500` etc. It reflects the intent better to the reader.
Also, would be nice to add the number of attempts to the name on line 50.

Reply:
Done.